dcos/cosmospackage.py

```diff
@@ -576,12 +600,18 @@ def _get_cosmos_header(request_name, version):
     :returns: dict of required headers
     :rtype: {}
     """
-
-    request_name = request_name.replace("/", ".")
-    return {"Accept": _get_header("{}-response".format(request_name),
-                                  version),
-            "Content-Type": _get_header("{}-request".format(request_name),
-                                        "v1")}
+    if request_name == "add":
+        return {
+            'Accept': 'application/vnd.dcos.package.add-response+json;charset=utf-8;version=v1',
+            'Content-Type': 'application/vnd.dcos.universe.package+zip;version=v1',
+            'Transfer-Encoding': 'chunked',
+        }
+    else:
+        request_name = request_name.replace("/", ".")
+        return {"Accept": _get_header("{}-response".format(request_name),
+                                      version),
+                "Content-Type": _get_header("{}-request".format(request_name),
+                                            "v1")}
```

Comment:
Not related to this PR but can you please fix the indentation to match your indentation style.